notebook_demo.ipynb

```diff
@@ -392,22 +364,15 @@
    "outputs": [],
    "source": [
     "from prompt2model.dataset_generator import OpenAIDatasetGenerator, DatasetSplit\n",
-    "from prompt2model.utils.logging_utils import get_formatted_logger\n",
-    "import logging, nest_asyncio\n",
-    "\n",
-    "nest_asyncio.apply()\n",
-    "\n",
-    "generator_logger = get_formatted_logger(\"DatasetGenerator\")\n",
-    "generator_logger.setLevel(logging.INFO)\n",
```

Comment:
Why delete the logger? 🤔

Reply:
Two reasons:
1. It was very verbose and output many logging lines
2. We want to keep the code in the Jupyter notebook as simple as possible so it doesn't overwhelm readers.